deployments/compose/README.md

```diff
@@ -35,3 +35,47 @@ Can access Trino using trino-cli, run under 'client' profile:
 docker compose --profile client run --rm trino-client
 ```
 
+### DBT
```

Comment:
If all the sections from here are part of '### DBT' make sure they got 4 or 5 '#' - in order to under they are sub-subjects